django/contrib/gis/templates/gis/openlayers.html

```diff
@@ -1,17 +1,11 @@
 {% load i18n l10n %}
-<style type="text/css">{% block map_css %}{% get_current_language_bidi as LANGUAGE_BIDI %}
-    #{{ id }}_map { width: {{ map_width }}px; height: {{ map_height }}px; }
-    #{{ id }}_map .aligned label { float: inherit; }
-    #{{ id }}_div_map { position: relative; vertical-align: top; float: {{ LANGUAGE_BIDI|yesno:"right,left" }}; }
-    {% if not display_raw %}#{{ id }} { display: none; }{% endif %}
-    {% endblock %}
-</style>
 
-<div id="{{ id }}_div_map">
-    <div id="{{ id }}_map"></div>
+<div id="{{ id }}_div_map" class="dj_map_wrapper">
+    <div id="{{ id }}_map" class="dj_map" data-width="{{ map_width }}" data-height="{{ map_height }}"></div>
```

Comment:
Do we need set `data-width`, `data-height`, and `style` on this `div`? :thinking:  It's already set on a `canvas`:
```html
<canvas style="width: 100%; height: 100%; display: block;" class="ol-unselectable" width="600" height="400"></canvas>
```
so it's probably redundant.

Reply:
Thanks, let's do this in a follow up commit.